docs/configuration-reference/components/azure-arc-onboarding.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Azure Arc onboarding configuration reference for Lokomotive
+weight: 10
+---
+
+## Introduction
+
+Azure Arc offers simplified management, faster app development, and consistent Azure services.
+
+With Azure Arc, you can:
+
+- Centrally manage a wide range of resources including
+[Windows](https://azure.microsoft.com/en-in/campaigns/windows-server/) and
+[Linux](https://azure.microsoft.com/en-in/overview/linux-on-azure/) servers, SQL server,
+[Kubernetes](https://azure.microsoft.com/en-in/services/kubernetes-service/) clusters and [Azure
+services](https://azure.microsoft.com/en-in/services/azure-arc/hybrid-data-services/).
+
+- Establish central visibility in the [Azure portal](https://azure.microsoft.com/en-in/features/azure-portal/)
+and enable multi-environment search with Azure Resource Graph.
+
+- Meet [governance](https://azure.microsoft.com/en-in/solutions/governance/) and compliance standards for
+apps, infrastructure and data with [Azure Policy](https://azure.microsoft.com/en-in/services/azure-policy/).
+
+- Delegate access and manage security policies for resources using role-based access control (RBAC) and [Azure
+Lighthouse](https://azure.microsoft.com/en-in/services/azure-lighthouse/).
+
+- Organise and inventory assets through a variety of Azure scopes, such as management groups, subscriptions,
```

Comment:
```suggestion
- Organise and inventory assets through various Azure scopes, such as management groups, subscriptions,
```

Reply:
Microsoft page on Azure arc has `a variety of`

Reply:
I added it since it is too wordy. Nothing right or wrong about it.